### PR TITLE
Melon's custom bottle

### PR DESCRIPTION
**Pull request by Melongod**
i hope i did this right

### CODE REVIEW

**Comment by warpdragon** [comment]
Wtf is that image. Does that even work in game?

**Comment by Melongod** [comment]
the image should work

File: game/scripts/vscripts/items/bottle.lua [hunk 1]
```diff
@@ -28,7 +28,8 @@ local special_bottles = {
   -- special people
   [110318967] = 4, -- timo
   [34314091] = 5, -- frej
-  [117027938] = 6
+  [117027938] = 6,
+  [370166341] = , -- Melon
```
[the remark this answers is not in the document]
Comment (Melongod) [reply game/scripts/vscripts/items/bottle.lua]: oops

**Comment by Melongod** [comment]
it's the same size as the other item tooltips, and it's a png file. i dont see why it wouldn't work

**Comment by imaGecko** [comment]
The failure actually originates from gravekeepers cloak lua

I wouldn't use the icon because all the custom bottles are actual bottles and looking at them you still understand them as bottles

**Comment by Haganeko** [comment]
This loses glance value.

**Comment by l34Um1** [comment]
I agree, this shouldn't be added. The bottles still need to look like bottles

**Comment by Yzanas** [comment]
yeah that's exactly what I told him as well

**Comment by l34Um1** [comment]
ye sure, just needs to look like a bottle

**Comment by Melongod** [comment]
sure

**Pull request merged**